packages/natds-icons/README.md

```diff
@@ -66,49 +79,64 @@ import {
 } from '@naturacosmeticos/natds-icons';
 ```
 
-For use on web environments, you will need to serve the fonts and consume the [CSS](https://github.com/natura-cosmeticos/natds-commons/blob/master/packages/natds-icons/dist/natds-icons.css) file as you see fit.
+Or like this:
 
-You can also import the [JSON file](https://github.com/natura-cosmeticos/natds-commons/blob/master/packages/natds-icons/dist/natds-icons.json) that contains the relationships between the icon names and their `unicode` counterparts (don't rely on the `unicodes`, they are generated automatically)
+```html
+<script src="node_modules/@naturacosmeticos/natds-icons/dist/natds-icons.json"></script>
 
-```jsx highlight-line="2"
-import {
-  NatdsIconsTtf,
-  iconNames
-} from '@naturacosmeticos/natds-icons';
+<link href="node_modules/@naturacosmeticos/natds-icons/dist/natds-icons.css" rel="stylesheet">
```

Comment:
esses imports não funcionam assim e acho que pode confundir.
acho legal seguir essa estrutura que ta no web de `with a module bundler vs in html`
https://github.com/natura-cosmeticos/natds-js/tree/main/packages/web#icons